recipes/server.rb

```diff
@@ -18,7 +18,7 @@
 #
 node.default['splunk']['is_server'] = true
 
-include_recipe 'chef-splunk::user'
+include_recipe 'chef-splunk::user' if !node['splunk']['server']['runasroot']
```

Comment:
I'd prefer to use the helper method, #splunk_runas_user, as part of this logic, like so:

```
include_recipe 'chef-splunk::user' unless splunk_runas_user == 'root'
```